src/frontend/.storybook/preview.js

```diff
@@ -1,3 +1,6 @@
+import {FC, useState} from "react";
```

Comment:
Does storybook support ts modules ? If yes, could you convert this js file into a ts file. In this way, eslint should lint this file.